pkg/node/types/node.go

```diff
@@ -134,7 +134,6 @@ func (n *Node) ToCiliumNode() *ciliumv2.CiliumNode {
 	}
 }
 
-
```

Comment:
Can be squashed with the previous commit.

Reply:
I lost the context for this. But in case this is for the first two commits (where `pkg/node/types/node.go`) is changed, I'll keep them separate until we get clean CI runs so that triaging will be easier if the IPAM bootstrap changes cause problems.

Reply:
Squashed the first two commits together.